### DISCUSSION TITLE
black screen

### DISCUSSION THREAD

**Discussion by moicced1** [hansschmucker/NVStreamer1080#15] [Q&A]
hello, 

thank you for what you do
However I have a question, I'm on a UWQHD screen (3440 x 1440 pixels).
When I go through your software, regularly I have a black screen and I have to restart my PC (I am in display port)
If I leave the screen on I have the impression that everything is working fine. is this normal?
thank you in advance for your help

**Comment by hansschmucker** [comment]
Hi moicced

That's actually a problem with the nvidia drivers. No idea why, but with DisplayPort you can't stream if the screen is off. It's actually the reason this program was created... so that I could switch to the HDMI dummy automatically ^^

--hans

**Reply by moicced1** [reply]
Thanks very much,
indeed in hdmi it works well ;)
However, I wonder what is the difference between the hdmi and the display port
